### ISSUE TITLE
SGTL5000 Audio Card Failures

### ISSUE THREAD

**Issue by K7MDL2** [K7MDL2/KEITHSDR#10]
I have burned though about 4 PJRC SGTL5000 Rev D audio cards (Teensy Audio backpack card).  I think most of these have been the same scenario.  My 4.3" RA8875 packaged into a small box along with the V2 Teensy motherboard and a RS-HFIQ.  On random 12V power ups the card is damaged. 12V feeds the RS-HFIQ and a 12 to 5V DC-DC converter for the Teensy board. Sometimes the failure pulls the Teensy 3.3V to near 0V.  Sometimes the audio chip is very hot.  Other times the Line In does not work but the Line out might.  This has not happened yet on my test bench setup where the RS-HFIQ is normally powered on and only the Teensy is power cycled or reset.  The 4.3" unit the reset/program switch is not normally accessible, so power cycling is used.   My best guess so far is there is a large voltage spike on the RS-HFIQ Line Out on power up killing the input circuit of the SGTL5000 line in pin.  Perhaps the timing affects this (12V comes on first due to a slight 5V delay through the DC-DC converter).

I plan to investigate this more with a digital storage o'scope, maybe catch a spike or power up sequence problem.

**Comment by mpvano** [comment]
Do you still have those audio cards? I have quite more qfp32 SGTL5000s here than I'm likely to ever use - since the QFP32 seems to be obsolete now.  I've switched my project (RadioHat) over to the 20 pin version - which is easily available. I'm located in MN so dropping a couple QFP32 in the mail is easy....

Mario (AE0GL)

**Comment by K7MDL2** [comment]
They are in a bag in the junk pile.  2or 3 I think left.  I have all the good ones I need.

I think you are suggesting you have extra bare chips in h right package type that could be used to replace the bad chips on those junk modules.  Interesting idea.  NO sure I will really ever need them.  I have been looking at replacing the SGTL5000 module with a much higher perf ADC/DAC.  Ideally one with built in PGA for convenience.

Mike K7MDL

**Comment by mpvano** [comment]
Yes, I still have about 20 left. As I said - my new boards use the 20 pin part...

Let me know - now or later on, I can drop them in the mail to you.

M

**Comment by K7MDL2** [comment]
I have been using the good old WM8960 module recently with Pico2 for a CW keyer and internet radio  projects.  Also the ESP32 with ES8311.

I have previously created a shim to use the WM8960 in the Teensy SDR.  It is not that much better for dynamic range but offered a headphone jack, mic inputs, and 1 W stereo speaker output.  

Decided effort was better spent on a high quality codec.  Never got there (so far).
